### PR TITLE
[ACR] Update sdk and prepare 4.0.0 release

### PR DESCRIPTION
**Pull request by ankurkhemani**
Generate SDK from latest swagger and prepare for Nuget release `4.0.0`

### CODE REVIEW

**Comment by ankurkhemani** [comment]
@erich-wang Once merged, I want the `4.0.0` sdk released to Nuget. Following steps [here](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#signing-and-publishing-nuget-packges), I was unable to join the `Azure SDK Release Partners` DG on idweb. Can you help me with this? Thanks!

**Comment by ankurkhemani** [comment]
// cc @djyou @jaysterp @northtyphoon

**Comment by jaysterp** [comment]
LGTM.

**Comment by erich-wang** [comment]
@ankurkhemani , could you please
1. Provide the swagger spec PR related to this change
2. Follow https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec to regenerate PR, e.g. I did not see eng\mgmt\mgmtmetadata\<rpName>.txt in this PR
3. Not quite understand your question about cannot join **Azure SDK Release Partners**, do you mean you cannot find it? By clicking the link, you'll be navigated to the group detail, send request by adding you as members, the owners will approve your request later.

**Comment by ankurkhemani** [comment]
@erich-wang 
1. I generated sdk using autorest on the [latest swagger spec](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/containerregistry/resource-manager):
`autorest readme.md --csharp --tag=package-2019-06-preview`
2. Updated sdk generation using `generate.cmd`. Can you please verify?
3. I was looking into DG(s) but it was a SG. Requested now.

File: src/SDKs/_metadata/containerregistry_resource-manager.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+2019-07-31 22:15:17 UTC
```
Comment (erich-wang) [inline src/SDKs/_metadata/containerregistry_resource-manager.txt]: this file should be put under **eng/mgmt/mgmtmetadata**

**Review by erich-wang — CHANGES_REQUESTED** [review]

File: src/SDKs/_metadata/containerregistry_resource-manager.txt (hunk 1 above)
Comment (ankurkhemani) [reply src/SDKs/_metadata/containerregistry_resource-manager.txt]: Done.

**Comment by ankurkhemani** [comment]
@erich-wang Can we please get this merged today? Thanks!

**Review by erich-wang — APPROVED** [review]

**Pull request merged**